src/it/resources/org/checkstyle/suppressionxpathfilter/descendanttoken/InputXpathDescendantTokenSwitchNoDefault.java

```diff
@@ -0,0 +1,14 @@
+package org.checkstyle.suppressionxpathfilter.descendanttoken;
+
+public class InputXpathDescendantTokenSwitchNoDefault {
+
+    void testMethod1() {
+        int x = 1;
+        switch (x) { // warn
```

Comment:
All 3 tests are the same by structure.

Please make them different:
Switch im method.
Switch nested to switch, violation on nested switch.
Switch in lambda.